### DISCUSSION TITLE
sunbird-rc-ui - needed more details about how to setup for a specific schema

### DISCUSSION THREAD

**Discussion by maheshkumargangula** [Sunbird-RC/community#102] [Issues]
@parthlawate  - I'm able to follow the sunbird-rc-core documentation and setup registry service with all the dependencies (with BloodDonor schema). 

I wanted to use the sunbird-rc-ui to connect to the registry API micro-service. The process running and showing the web page. But, there are no details or documentation about how to enable the forms to self-signup or register and login and other use cases. 

Please help with the documentation to setup UI along with the API Microservices.

cc: @coolbung

**Comment by parthlawate** [comment]
@code-blooded-guy Can you help here please ?

**Reply by code-blooded-guy** [reply]
Sure, I will connect with Mahesh.

**Comment by parthlawate** [comment]
since we want to encourage open communication, please make sure that as an output you reply here with as much detail as you can and also update documentation.

**Comment by code-blooded-guy** [comment]
Sure, updated the docs creating PR for it.
